addons/bus/websocket.py

```diff
@@ -314,8 +319,8 @@ def onclose(cls, func):
     def subscribe(self, channels, last):
         """ Subscribe to bus channels. """
         self._channels = channels
-        if self._last_notif_sent_id < last:
-            self._last_notif_sent_id = last
+        if self._min_notif_id < last:
+            self._min_notif_id = last
```

Comment:
What was the goal of reseting last on subscribe again?
Maybe we could just set the value only if `_min_notif_id` is 0?

Reply:
We talked about it yesterday, to me we should let the socket keep track of the id once it started. So i'd say we can indeed avoid setting it if already settled (would only be done on first subscribe)